app/assets/javascripts/components/overview/details.jsx

```diff
@@ -222,10 +222,6 @@ const Details = React.createClass({
         </div>
         <div className="module__data extra-line-height">
           {instructors}
-          {online}
-          {campus}
-          {staff}
-          {school}
```

Comment:
Here, I could probably remove all the computation done earlier to display this information as well. Should I do that?